core/src/main/scala/sttp/openai/OpenAi.scala

```diff
@@ -475,6 +487,7 @@ private object OpenAIEndpoints {
   val VariationsImageEndpoint: Uri = ImageEndpointBase.addPath("variations")
 
   def deleteFileEndpoint(fileId: String): Uri = FilesEndpoint.addPath(fileId)
+  def deleteFineTuneModel(model: String): Uri = ModelEndpoint.addPath(model)
```

Comment:
**suggestion:** `ModelEndpoint.addPath(model)` doesn't imply that it's a "delete" endpoint, only that it's a "modelEndpoint". Actually, similar remark applies to other methods here.

Reply:
it seems like it is not an endpoint but more like an URI so shall we rename it to `deleteFineTuneModelUri` then?

Reply:
yeah this sounds good :)